modules/sdk-coin-avaxp/src/lib/delegatorTxBuilder.ts

```diff
@@ -302,7 +324,6 @@ export class DelegatorTxBuilder extends TransactionBuilder {
         )
       )
     );
-
-    return { inputs, outputs };
+    return { inputs, outputs, credentials: credentials.length == 0 ? this.transaction.credentials : credentials };
```

Comment:
nit
```suggestion
    return { inputs, outputs, credentials: credentials.length === 0 ? this.transaction.credentials : credentials };
```

Reply:
Sorry for this, I saw it now! It will be fix later